configs/generic_config.txt

```diff
@@ -1,4 +1,5 @@
 # vfat filesystem:
+# TODO: vfat is going to be deprecated for testing in favour of 9p
```

Comment:
remove this

we still want vfat for testing diskbooting, e.g.

Reply:
done